pykusto/expressions.py

```diff
@@ -96,6 +96,13 @@ def has(self, exp: StringType) -> 'BooleanExpression':
         # The pattern for the search expression must be a constant string.
         return BooleanExpression(KQL(f'{self.kql} has {to_kql(exp)}'))
 
+    def has_cs(self, exp: StringType) -> 'BooleanExpression':
```

Comment:
Why is this a member of `BaseExpression`, and not `StringExpression`?

Reply:
because has was, but I guess you are right, I can move them both to StringExpression

Reply:
Indeed this was a mistake with `has`. Also I think it is more consistent with other pykusto methods to unify `has` and `has_cs` to one method. Continued in #109